src/main/java/eggventory/parsers/ParseDelete.java

```diff
@@ -45,15 +35,27 @@ public Command parse(String inputString) throws InsufficientInfoException, BadIn
         switch (deleteInput[0]) {
 
         case "stock":
+            if (!Parser.checkIfCommandComplete(inputString, 1)) {
```

Comment:
technically wouldn't the naming convention for the method be like "commandIsComplete" lol

Reply:
isCommandComplete?

Reply:
yeah smth like that idk why it would say "check" inside
just standard boolean method naming